pages/pipelines/command_step.md

```diff
@@ -300,9 +300,36 @@ _Optional Attributes_
   <tr>
     <td><code>exit_status</code></td>
     <td>
-      The exit status number that will cause this job to retry ('*' does not include 0) <br>
-      <em>Example:</em> <code>"*"</code><br>
-      <em>Example:</em> <code>2</code>
+      The exit status number that causes this job to retry ('*' does not include 0) <br>
+      <p><em>Examples:</em></p>
+      <ul>
+        <li><code>"*"</code></li>
+        <li><code>2</code></li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td><code>signal</code></td>
+    <td>
+      The signal that causes this job to retry. This signal only appears if the agent sends a signal to the job and an interior process does not handle the signal. <code>SIGKILL</code> propagates reliably because it cannot be handled, and is a useful to differentiate graceful cancelation and timeouts.
```

Comment:
Oops, I think this was missed in the last update. I assume it's either meant to be:

```suggestion
      The signal that causes this job to retry. This signal only appears if the agent sends a signal to the job and an interior process does not handle the signal. <code>SIGKILL</code> propagates reliably because it cannot be handled, and is a useful way to differentiate graceful cancelation and timeouts.
```

Or:

```suggestion
      The signal that causes this job to retry. This signal only appears if the agent sends a signal to the job and an interior process does not handle the signal. <code>SIGKILL</code> propagates reliably because it cannot be handled, and is useful to differentiate graceful cancelation and timeouts.
```